src/css/pages/home-page.css

```diff
@@ -0,0 +1,76 @@
+.home-page {
+  background: url("../assets/images/background-1.png") repeat;
+}
+
+.home-page__title {
+  width: 41.6%;
+}
+
+.home-page__subtitle {
+  color: #4b4e4c;
+  font-size: 20px;
+  margin: 0 auto;
+  text-transform: uppercase;
+  width: 40%;
+}
+
+.home-page__subtitle-ribbon {
+  position: relative;
+}
+
+.home-page__subtitle-ribbon::before,
+.home-page__subtitle-ribbon::after {
+  content: "\2022 \2001 \2022 \2001 \2022 \2001 \2022 \2001";
+  position: absolute;
+  z-index: 1;
+
+  color: #fff1ce;
+  text-shadow: 0.1rem 0.1rem 0.1rem rgba(0, 0, 0, 0.2);
+}
+
+.home-page__subtitle-ribbon::before {
+  top: 1.3em;
+  left: -6.5em;
+}
+
+.home-page__subtitle-ribbon::after {
```

Comment:
This block is equal for `.home-page__subtitle-ribbon::before` and `.home-page__subtitle-ribbon::after` declarations, so it should be moved to the common one.